tests/rptest/services/flink.py

```diff
@@ -22,12 +22,52 @@
 instance_specs = {
     "default": {
         "vcpus": 2,
-        "ram": 16,
+        "ram": 16
     },
     "i3en.xlarge": {
         "vcpus": 4,
-        "ram": 32,
-    }
+        "ram": 32
```

Comment:
nit: I think this should belong in some util class that maintains a list of known instance types and configurations.